includes/discovery/sensors/count/fortigate.inc.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/*
+ * LibreNMS Dantel Webmon generic sensor
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ *
+ * @package    LibreNMS
+ * @subpackage webui
+ * @link       http://librenms.org
+ * @copyright  2019 LibreNMS
+ * @author     LibreNMS Contributors
+ */
+
+$session_rate = [
+    'Sessions/sec 10m avg' => '.1.3.6.1.4.1.12356.101.4.1.12.0',  //FORTINET-FORTIGATE-MIB::fgSysSesRate10.0
+    'Sessions/sec 60m avg' => '.1.3.6.1.4.1.12356.101.4.1.14.0',  //FORTINET-FORTIGATE-MIB::fgSysSesRate60.0
+];
+
+$index = 0;
+foreach ($session_rate as $descr => $oid) {
+    $result = snmp_get($device, $oid, '-Ovq');
+    $result = str_replace(' Sessions Per Second', '', $result);
+
+    discover_sensor(
+        $valid['sensor'],
+        'count',
+        $device,
+        $oid,
+        $index,
```

Comment:
Your indexing method is not correct. It will not be stable if one sensor appears between 2 existing ones. The index should be built out of the OID table name AND the SNMP index (which you can expect to be stable).

Reply:
Thanks, I corrected it. Hope it's ok now.